tests/plan/lint/test.sh

```diff
@@ -4,26 +4,36 @@
 
 rlJournalStart
     rlPhaseStartSetup
-        rlRun "tmp=\$(mktemp -d)" 0 "Creating tmp directory"
-        rlRun "pushd $tmp"
+        rlRun "pushd data"
         rlRun "set -o pipefail"
         rlRun "tmt init"
-        rlRun "tmt plan create --template mini good"
-        rlRun "echo 'execute:' > bad.fmf"
     rlPhaseEnd
 
     rlPhaseStartTest "Good"
         rlRun "tmt plan lint good"
+        rlRun -s "tmt plan lint valid_fmf"
+        rlAssertGrep "pass fmf remote id is valid" $rlRun_LOG
     rlPhaseEnd
 
     rlPhaseStartTest "Bad"
-        rlRun "tmt plan lint bad | tee output" 1
-        rlAssertGrep 'fail execute step must be defined' output
-        rlAssertGrep 'warn summary is very useful' output
+        rlRun -s "tmt plan lint bad" 1
```

Comment:
$rlRun_LOG should be removed manually each time - Every rlRun -s creates a new file

Reply:
looking

Reply:
Yes, because of this I'd say `command | tee output` is still a better option.

Reply:
You mean `command 2>&1 |tee output` :)  And for exactly this mixing stdout/stderr I'd say it is easier to use `-s` even when you need to delete the file afterwards.

Reply:
I added removal of those files. I would keep `$rlRun_LOG`